_includes/header.html

```diff
@@ -1,15 +1,19 @@
-<header class="header">
-    <div>
-        <picture class="logo" >
-            <source srcset="{{ site.baseurl}}/assets/img/logo.webp" type="image/webp">
-            <source srcset="{{ site.baseurl}}/assets/img/logo.png" type="image/png"> 
-            <img alt="" src="{{ site.baseurl}}/assets/img/logo.png">
-        </picture>
-        <picture class="logo" >
-            <source srcset="{{ site.baseurl}}/assets/img/SPACESTATION3D.webp" type="image/webp">
-            <source srcset="{{ site.baseurl}}/assets/img/SPACESTATION3D.png" type="image/png"> 
-            <img alt="" src="{{ site.baseurl}}/assets/img/SPACESTATION3D.png">
-        </picture>
+<center>
```

Comment:
This tag isn't recommended in HTML5. Should instead add a style (text-align: center) to the "header" class in the css.

Reply:
I didn't even need this at all... -_-